torchtext/datasets/enwik9.py

```diff
@@ -50,4 +50,4 @@ def EnWik9(root: str, split: Union[Tuple[str], str]):
     )
 
     data_dp = FileOpener(cache_decompressed_dp, mode="b")
-    return data_dp.readlines(decode=True, return_path=False)
+    return data_dp.readlines(decode=True, return_path=False, strip_newline=False)
```

Comment:
Usually, it is good to strip the new lines at the end of each line. Can we change this going forward?

Reply:
I know it breaks BC, but does it matter as we are revamping the library?

Reply:
> Usually, it is good to strip the new lines at the end of each line. Can we change this going forward?

It's a good point, I am not so sure why it is like that when they were first added. In-fact now when I think about it, it's not even consistent across our datasets. 

> I know it breaks BC, but does it matter as we are revamping the library?

Well, we wanted the migration to be not BC breaking. But if we do end up doing so, let's make sure to note all the BC breaking changes in release notes.

Reply:
> Well, we wanted the migration to be not BC breaking. But if we do end up doing so, let's make sure to note all the BC breaking changes in release notes.
> 
> Edit: Also would be great not to mix the BC breaking changes with the migration.

I also updated the SST2 dataset to return a `int` for the labels even though it was returning strings before because that seemed like the expected behavior. Does it matter whether we do it in the same PR as the migration vs a different PR if we note all the BC breaking changes in the PR summary?

Reply:
Does it matter whether we do it in the same PR as the migration vs a different PR if we note all the BC breaking changes in the PR summary?

In general, it's a good practice to logically separate PRs that makes it easy for tracking purpose as well as to review :)